ARM Templates/FailoverReprotect/azuredeploy.parameters.json

```diff
@@ -0,0 +1,66 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "identity": {
+           "value": "GEN-UNIQUE-32",
+          "metadata": {
+              "description": "ARM ID of a user managed identity that has contributor access to the resource group being deployed."
+          }
+      },
+      "vaultSubscriptionId": {
+           "value": "GEN-UNIQUE-32",
```

Comment:
shouldnt these be blank?

Reply:
These are the standard recommended placeholders for ARM Templates.